Apps/phcuckoo/view_results.html

```diff
@@ -78,18 +78,24 @@
   text-align: left;
 }
 
+pre{
```

Comment:
I think we do not advice to have custom CSS in the html pages. Additional input from rest of team suggested.

Reply:
We usually keep this practice in the readme file. And, we haven't added that tag in output, we have just adjusted the color and I guess we are not following that practice in the custom output view of the action.

Reply:
I agree with Alexa and custom CSS should be removed

Reply:
Can we remove it then?

Reply:
In this scenario, the custom output view is not getting displayed properly in the dark theme. We are not able to see properly the response text in the custom output view due to the dark color of the text. So, we have defined this custom CSS but if not the standard practice to use custom CSS in the tag, then, should we add inline CSS in the corresponding response text? Because at the end we have to display proper custom output view which is easy to read.

Reply:
Without using inline CSS for color
![image](https://user-images.githubusercontent.com/72184309/105501263-fefc1200-5ce9-11eb-8dc6-10db5c18a268.png)
Using inline CSS for color
![image](https://user-images.githubusercontent.com/72184309/105501464-4387ad80-5cea-11eb-8696-92ea160fb7e4.png)

Due to less visibility of output data in dark theme, we have added inline CSS and updated text color to make it readable in both the themes.

Reply:
I see. How does it look with light theme?

Reply:
![image](https://user-images.githubusercontent.com/57654567/105534025-e6542200-5d12-11eb-960e-bdac5d824d0e.png)

Reply:
Thank you. I think it is fine to keep it in for now, but please create a Jira PPS bug for this so that we don't have to be modifying the CSS in our apps in the future.

Reply:
@mjiyani-crest can you please link the PPS bug?

Reply:
Yes, I will create and link it here.

Reply:
@mjiyani-crest Can you please link it?